flink-ml-core/src/main/java/org/apache/flink/ml/common/utils/ComparatorAdapter.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.ml.common.utils;
+
+import org.apache.flink.api.common.typeutils.TypeComparator;
+
+import java.util.Comparator;
+
+/** Utility class to convert {@link TypeComparator} to a {@link Comparator}. */
+public class ComparatorAdapter {
+
+    /**
+     * Adapts a {@link TypeComparator} to a {@link Comparator}.
+     *
+     * @param typeComparator The input typeComparator.
+     * @param <T> The class type of the input element.
+     * @return The converted comparator.
+     */
+    public static <T> Comparator<T> getComparator(TypeComparator<T> typeComparator) {
```

Comment:
This method is only called in one place for now and its body has just one line. It seems a bit overkill to have one class/file/method for this one line.

Reply:
Thanks for the comments. I have removed this class.